paper.md

```diff
@@ -118,7 +118,7 @@ due to its composability, automatic differentiation and support for GPUs/TPUs, a
 the NumPy [@numpy] API resulting in a low barrier to entry.
 
 `qujax` is a lightweight, purely functional library written entirely in JAX, 
-composing seamlessly with the ever-expanding JAX ecosystem (e.g. [@deepmindjax; @blackjax; @mocat]). 
+composing seamlessly with the ever-expanding JAX ecosystem [@deepmindjax; @blackjax; @mocat]. 
```

Comment:
```suggestion
composing seamlessly with the ever-expanding JAX ecosystem, e.g. @deepmindjax, @blackjax, @mocat. 
```
How about this? keep e.g. but no brackets?
I agree people do not expect reference lists to be complete but this list is very much not complete (JAX ecosystem is vast) and therefore we highlighted a few that might be particularly relevant for composition with qujax.

Reply:
@lucydot Do you think Sam's suggestion is a good way to resolve this issue? If so we are happy to go ahead and merge it!

Reply:
I don’t think this is the correct formatting for a list of references in markdown; for example, `2020` is listed twice in blue as an in-line reference to different sources. I have always seen prior: `[@deepmindjax; @blackjax; @mocat]` which formats the reference list consistent with style used for single references. 

Perhaps we could zoom out a bit and ask what is the need for such references if they are random examples from a very large eco-system? If they are not random, and have some particular relevance to the project then perhaps this could be used to frame the list context, otherwise they might not be required at all?

Reply:
Yeah ok I'm happy to remove the references, since qujax is very general and the JAX ecosystem is very large there are many different use cases and combinations (which is a great strength!), but we don't necessarily need to prompt the reader in terms of these directions. I think without the references it is clear that the JAX ecosystem is broad and composable.

Reply:
Yup I agree @gamatos!